### PR TITLE
Microverse Project Floating and Positioning: New York Times Article Copy

### PR DESCRIPTION
**Pull request by Janluc**
## New York Times Article Clone
- Used Float, Grid, and Flexbox for the general styling of the page.
- Use of Navbar and Footer
- Use of Article, Sections, and other semantic HTML

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,57 @@
...
+# Positioning and Floating Elements
+
+A clone of a New York Times Article page, made using HTML and CSS and implementing all positioning elements such as Float, Flexbox, and Grid. It includes an Article section, Navbar, Footer, and various spans.
+
 
-> One paragraph statement about the project.
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
```
Comment (kevcap) [inline README.md]: - [ ] The description is above the image, so you can remove this text.

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️


#### Overall the project looks really nice and I can see that you guys are working hard on it, and don't worry, you're almost done, there are just some required changes that you guys have to implement to proceed:


_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.

## [REQUIRED CHANGES]
- [x] Your project's navbar should have an embedded NYT logo and it should be a link.
- [x] At the section "The Theory of Inflation", the image should be only one with text above it. (Just like the original)
- Yours (8 images)
image
- Original (Only one image)
image

- [x] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
       7 |   color: black;
       7 |   text-decoration: none;
       6 |   font-size: 12px;
       6 |   margin: 0 auto;
       5 |   color: #666;
       5 |   color: #888;
       5 |   justify-content: space-between;

## [OPTIONAL CHANGES]
- [ ] Please use the images of the actual page, instead of that grey ones;

### Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question about this review? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

-[ ] Your project's navbar should have an embedded NYT logo and it should be a link.nPlease make the  New York Times logo a link. 

-[ ] In the section "The Theory of Inflation", the image should be only one with text above it. (Just like the original)
-[ ] Your page is not correctly styled. Please work on it.

chrome-capture (15)



**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## Awesome work guys! :clap: :clap: :clap:

:100: The sites looks very close to the original!
:100: Using semantic HTML
:100: Great improvement with the applied changes

You're almost there, please work on the issues below:

## Required Changes
### [Mandatory]

- [ ] NewYorkTime logo not centered
image

- [ ] Section not displaying correctly
image

- [ ] **Outdated link to live preview. Fixing this should solve the issues above.**

- [ ] Issue, MIT licence links broken.
image

- [ ] ```footer``` should be outside the ```main``` tag.

### [OPTIONAL]

- [ ] Images not showing. Try to make the project looks as similar as possible. This is a great mockup but the project should be a clone.
image
image
image

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: Hi guys! 

Please ensure the project pass all linter checks before submitting a re-review.

- [ ] Project not passing all linters check.

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Congrats team!! you've worked on all required changes!

You may now merge to master!

## :fireworks: Well done!

**Pull request merged**